packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -296,3 +371,72 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     });
   }
 }
+
+class ErrorNavigator extends StatelessWidget {
+  const ErrorNavigator(
+      {Key key,
+      @required this.selectedErrorIndex,
+      @required this.errorCount,
+      @required this.onSelectedErrorIndexChanged})
+      : super(key: key);
+
+  final int selectedErrorIndex;
+  final int errorCount;
+  final void Function(int) onSelectedErrorIndexChanged;
+
+  @override
+  Widget build(BuildContext context) {
+    final label = selectedErrorIndex != null
+        ? 'Error ${selectedErrorIndex + 1}/$errorCount'
+        : 'Errors: $errorCount';
+    return Container(
+      color: devtoolsError,
+      child: Padding(
+        padding: const EdgeInsets.symmetric(horizontal: 12, vertical: 6),
```

Comment:
instead of using 12 and 6 for const padding values throughout this code, can we use the padding and spacing consts from theme.dart? defaultSpacing, denseSpacing, densePadding, etc.